Competitive Programming/c2cpp.md

```diff
@@ -0,0 +1,117 @@
+# C to Cpp for competitive
+
+This article is written for specifically keeping competetive in mind and not for using cpp in development or any other context. Also this aims at getting started to use good and important features of c++ as quickly as possible so skips many details.
```

Comment:
`s/is written for specifically/was written/`
`s/competetive/competitive programming` (mind your spelling).
`s/cpp/C++`

It also aims to get you started with good and important features of C++ as quickly as possible, so skips many details.